spark-jobs/src/test/scala/filodb/downsampler/DownsamplerMainSpec.scala

```diff
@@ -67,8 +67,9 @@ class DownsamplerMainSpec extends FunSpec with Matchers with BeforeAndAfterAll w
   val lastSampleTime = 1574273042000L
   val downsampler = new Downsampler
 
-  //Run for current hour for testing, actual job migrates previous hour's index updates
-  val indexUpdater = new IndexJobDriver(hour())
+  //Index migration job, runs for current 2hours for testing. actual job migrates last 6 hour's index updates
+  val currentHour = hour()
+  val indexUpdater = new IndexJobDriver(currentHour - 2, currentHour)
```

Comment:
This may be flaky. If index was written at 9:59:59.999 then test may fail.

Reply:
test run is for current and previous hour, so it should pick both 9th and 10th hour updates